website/docs/components/table/advanced-table/partials/accessibility/accessibility.md

```diff
@@ -14,6 +14,64 @@ The table row element cannot receive interactions, meaning actions cannot be att
 
 Unlike the Table component, each cell receives focus in the Advanced Table to support users navigating through the table efficiently with a keyboard. For any other interactions, you must use interactive elements (buttons, links, etc.) within the cells. 
 
+## Keyboard Navigation
+
+The Advanced Table supports two keyboard interaction modes: Navigation Mode and Action Mode. These modes allow users to move through table data and interact with elements using only a keyboard.
+
+### Navigation Mode
+
+When a user tabs into the Advanced Table, the first cell is focused. Arrow keys move between cells.
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+![Keyboard focus on the sortable 'Artist' column header with sorting controls active.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-tab.png =402x*)
+
+<Doc::Badge @type="neutral">Arrow key Down</Doc::Badge>
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-arrow-key-down.png =402x*)
+
+**Home (fn + left)**
+Move focus to the first cell in the row.
+
+**End (fn + right)**
+Move focus to the last cell in the row.
+
+**Page Up (fn + up)**
+Move focus to the first cell in the column.
+
+**Page Down (fn + down)**
+Move focus to the last cell in the column.
+
+**Horizontal Scrolling**
+If the table extends beyond the screen, arrow keys allow movement to access hidden columns.
+
+### Action Mode
+
+Press Enter in Navigation Mode to enter Action Mode and interact with elements inside a cell.
+
+<Doc::Badge @type="neutral">Enter</Doc::Badge>
+
+![Keyboard focus on 'The Beatles' link inside the table.](/assets/components/table/advanced-table/advanced-table-keyboard-action-mode-link.png =402x*)
+
+**Enter**
+- If the cell contains one interactive element, focus moves to that element instead of activating it.
+- If multiple interactive elements exist, focus moves to the first one. Use Tab to navigate between them.
```

Comment:
Q: Should "Tab" be capitalized here? Maybe we could say "the tab key" instead here. (to keep it distinct from the Tabs/Tab components which do use capitalization.)